ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -813,6 +813,70 @@
         ]
       }
     },
+    {
+      "name": "stream-stream-stream - outer-outer",
+      "statements": [
+        "CREATE STREAM S1 (ID INT KEY, V0 bigint) WITH (kafka_topic='left', value_format='JSON');",
```

Comment:
Why are names (`ID`, `V0`) is upper case, while keywords (`BIGINT`) are in lower case?

Reply:
Pretty much all the tests in this file are copy-pasted from each other, and they're all like this. I've updated the casing for the new tests, but updating the casing for all the existing teams is out of scope IMO.

Reply:
Absolutely. I did not ask to rewrite existing tests.